travis/test_server.py

```diff
@@ -271,7 +271,7 @@ def hidden_line(line, main_modules, addons_path_list=None,
      - Hidden "warning no translation" if the main lang file exists.
     """
     lang_regex = re.compile(
-        r": module (?P<module>\w+): no translation for language (?P<lang>\w+)")
+        b": module (?P<module>\w+): no translation for language (?P<lang>\w+)")
```

Comment:
br""

Reply:
I dont know this way, thanks for teach me